### PR TITLE
Ask answer streamfield

### PR DESCRIPTION
**Pull request by virginiacc**
Moves answer page content into streamfield blocks and makes other updates to the answer page layout.

## Additions

- Migration to move answer content into streamfield
- Blocks and templates for tips and answer content

## Removals
- Remove separate `search-bar.html` organism since it was only being used for Ask

## Changes
- Update answer page layout
- Move content into streamfield blocks

## Testing

1. Check out branch
2. Run migration & `gulp styles`
3. Navigate to an English and Spanish answer page. Ensure its content still displays correctly, the search bar works, and the style/layout matches screenshot.
4. Add a tip to the content of an answer page and verify that it displays as in design.

## Screenshots

answer-page

## Design
22fa5280-45ad-11e9-8010-e61f48f7d896

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: cfgov/ask_cfpb/models/pages.py [hunk 1]
```diff
@@ -678,7 +686,7 @@ class AnswerPage(CFGOVPage):
 
     edit_handler = TabbedInterface([
         ObjectList(content_panels, heading='Content'),
-        ObjectList(sidebar_panels, heading='Sidebar (English only)'),
+        ObjectList(sidebar_panels, heading='Sidebar'),
```
Comment (richaagarwal) [inline cfgov/ask_cfpb/models/pages.py]: 😃

File: cfgov/unprocessed/css/pages/ask.less [hunk 2]
```diff
@@ -105,6 +104,99 @@
 /* Answer page */
 
 .ask-cfpb-page__answer {
+    .respond-to-min( @bp-lg-min, {
+        .content_sidebar {
+            padding-top:0;
+        }
+    });
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/pages/ask.less]: TRIVIAL: mix of spaces and not spaces between parens and colons here and throughout.

File: cfgov/unprocessed/css/pages/ask.less [hunk 3]
```diff
@@ -105,6 +104,99 @@
...
+
+    .o-expandable_cue-close {
+        display: none;
+    }
+
+    .answer-text {
+        h2,
+        h3,
+        h4 {
+            margin-top: 30px;
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/pages/ask.less]: Mix of hardcoded margins and `@grid_gutter-width` here and throughout. Looks like this should be `margin-top: @grid_gutter-width;`.

File: cfgov/unprocessed/css/pages/ask.less (hunk 2 above)
Comment (virginiacc) [reply cfgov/unprocessed/css/pages/ask.less]: Should be fixed with latest commits.

File: cfgov/unprocessed/css/pages/ask.less (hunk 3 above)
Comment (virginiacc) [reply cfgov/unprocessed/css/pages/ask.less]: Should be fixed with latest commits.

**Comment by virginiacc** [comment]
@higs4281 We explored adding another formatting option for warnings, but I think for now they're just going to be implemented as normal text with a heading.

File: cfgov/jinja2/v1/ask-cfpb/answer-page.html [hunk 4]
```diff
@@ -27,18 +27,22 @@
 {%- endblock %}
 
 {% block content_main %}
+<div {% if flag_enabled('HOW_TO_SCHEMA', request) %}
```
[the remark this answers is not in the document]
Comment (virginiacc) [reply cfgov/jinja2/v1/ask-cfpb/answer-page.html]: @anselmbradford I think whitespace before the closing bracket of a tag is valid. It is nicer without the space, though -- I have another answer page PR coming up so I can make these spacing changes there.

**Pull request merged**